src/borg/archiver.py

```diff
@@ -1222,6 +1226,116 @@ def do_debug_dump_archive_items(self, args, repository, manifest, key):
         print('Done.')
         return EXIT_SUCCESS
 
+    def debug_json(self, d):
```

Comment:
the code in here is not directly related to json, you just use its result for json generation.

so, just call it prettify_dict()?